SmilesMilesTests/SmilesMilesTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SmilesMilesTests.swift
+//  SmilesMilesTests
+//
+//  Created by Pavan Sai Nallagoni on 2/8/23.
+//
+
+import XCTest
+@testable import SmilesMiles
+
+final class SmilesMilesTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        // Any test you write for XCTest can be annotated as throws and async.
+        // Mark your test throws to produce an unexpected failure when your test encounters an uncaught error.
+        // Mark your test async to allow awaiting for asynchronous code to complete. Check the results with assertions afterwards.
+    }
+
+    func testPerformanceExample() throws {
+        // This is an example of a performance test case.
+        self.measure {
+            // Put the code you want to measure the time of here.
+        }
+    }
```

Comment:
Why add this code if you don't have any unit tests.  It's a good idea to have unit tests though.  For that you would need to refactor your code for your views to use a view model.